### ISSUE TITLE
Change response in Create declaration request

### ISSUE THREAD

**Issue by DPashchenko** [Nebo15/ehealth.api#527]
Change declaration request response according to specification.
Apiary [specification](http://docs.ehealthapi1.apiary.io/#reference/public.-medical-service-provider-integration-layer/declaration-requests-tbd/create-declaration-request)

**Comment by gmile** [comment]
Done in #554

**Comment by DPashchenko** [comment]
Не соответствует:
```
Employee{
+party
}
legal_entity{
+licenses
+accreditation
}
divisions{
-status
-mountain_group
-location
}```

**Comment by gmile** [comment]
Done in 12d3a2f3dd9751058c8301789f56e4d036701147 and 11591440a34a6318d46036ac798608c939d0c3d3. Available since 0.3.59 and already deployed.
